src/promptflow-devkit/promptflow/_sdk/_tracing.py

```diff
@@ -441,7 +400,10 @@ def start_trace_with_devkit(collection: str, **kwargs: typing.Any) -> None:
     # print tracing url(s) when run is specified
     _print_tracing_url_from_local(pfs_port=pfs_port, collection=collection, exp=exp, run=run)
     if ws_triad is not None and is_azure_ext_installed:
-        _print_tracing_url_from_azure_portal(ws_triad=ws_triad, collection=collection, exp=exp, run=run)
+        print(
+            "You can view the traces in azure portal. The portal link will be printed once the run is finished "
```

Comment:
The message should first say sth like: 

trace.destination is configured to tracing to azure. The portal link will be printed once the run is finished...

Reply:
Added the trace destination info. Didn't start with `Trace destination is set to xxx` because it looks confusing while the last line just printed a local trace link.